vaccine_feed_ingest/runners/az/pinal_ph_vaccinelocations_gov/normalize.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest_schema import location as schema
+
+logger = logging.getLogger(__name__)
+
+RUNNER_ID = "az_pinal_ph_vaccinelocations_gov"
+
+
+def _get_id(site: dict) -> str:
+    id = f"{_get_name(site)}_{_get_city(site)}".lower()
+    id = id.replace(" ", "_").replace(".", "_").replace("\u2019", "_")
+    id = id.replace("(", "_").replace(")", "_").replace("/", "_")
+    return id
+
+
+def _get_name(site: dict) -> str:
+    return site["providerName"]
+
+
+def _get_city(site: dict) -> str:
+    return site["city"].lstrip().rstrip()
+
+
+# address is loosely structured and inconsistent, so we're going to bash our
+# way through it, mostly parsing from the end of the string
+def _get_address(site: dict) -> Optional[schema.Address]:
+    if "address" not in site or not site["address"]:
+        return None
+
+    address = site["address"]
+    address = address.replace("        ", " ")
+    address = address.replace("    ", " ")
+    address = address.replace("    ", " ")
+    address = address.replace("  ", " ")
+    address = address.replace(" ,", ",")
+    address = address.replace(",,", ",")
+    address = address.lstrip()
+    address = address.rstrip()
```

Comment:
Could this be simplified to something like the following (to guard against future changes in spacing)?
```suggestion
    address = re.sub("\s+", " ", address)
    address = re.sub("\s*,+", ",", address)
    address = address.strip()
```